app/controllers/users.js

```diff
@@ -65,11 +67,12 @@ exports.signin = (request, response, next) => {
       if (user != null) {
         bcrypt.compare(userLogin.password, user.password).then(valid => {
           if (valid) {
-            const NewSession = {
-              accessToken: sessionManager.generateAccessToken
-            };
-            response.status(200);
-            response.send(NewSession);
+            sessionManager.generateAccessToken(user).then(accessToken => {
+              response.status(200);
```

Comment:
And here i think that with the send is enough, because the default status is 200, but feel free to confirm this